include/no_os_print_log.h

```diff
@@ -55,54 +55,81 @@
 #define NO_OS_LOG_LEVEL NO_OS_LOG_INFO
 #endif
 
+#if defined(PRINT_TIME)
+#define pr_time			{ struct no_os_time _t = no_os_get_time(); \
+			printf("[%5d.%06d] ", _t.s, _t.us); }
```

Comment:
did astyle choose this indentation here ?

Reply:
I modified the indentation and run astyle in the new version.